src/GraphQL/Types/Schema.cs

```diff
@@ -25,7 +25,7 @@ public interface ISchema : IDisposable
 
     public class Schema : ISchema
     {
-        private readonly Lazy<GraphTypesLookup> _lookup;
+        protected readonly Lazy<GraphTypesLookup> _lookup;
```

Comment:
I do want to have a way to provide additional types, though I'm not sure this is the way I want to support it.  I was originally thinking of having a constructor overload that took in an `IEnumerable<Type>` or some sort of loader.